issues/iotimeout.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: doc
+sublayout: issue
+title: Docker is returning an i/o timeout
+frontpage: True
+---
+Attemping to interact with a remote repository yields an `i/o timeout`.
+
+```
+$ sudo docker pull ...
+FATA[0021] Error response from daemon: v1 ping attempt failed with error: Get https://quay.io/v1/_ping: dial tcp: i/o timeout.
+```
+
+Please read over and post in [this docker issue](https://github.com/docker/docker/issues/13337) which is tracking the bug.
```

Comment:
"This is a known issue currently in Docker. To track the progress of this issue, please see (link here). Any additional feedback is also appreciated on that issue. "